content/docs/start/model-management/model-cicd.md

```diff
@@ -0,0 +1,179 @@
+
+```bash
+dvc get https://<path-to-my-git-forge-account>/example-get-started-experiments models/model.pth --rev v1.0.0
+```
+
+where you just need to replace `<path-to-my-git-forge-account>` with your
+GitHub/GitLab/Bitbucket account path. This will download the model version
+v1.0.0 to the location from where you are calling the `dvc get` command.
+
+If you're using Studio this is even easier. Go to the Models dashboard, click on
+the three dots next to the name of your model and select "Download model file".
+Then you can select the model version you're interested in and Studio will
+generate the correct call of the `dvc get` command for you. You then just need
+to copy it and run it where you want the model to be downloaded. You can see all
+the steps here:
+
+![Downloading models](/img/mr-download-model.gif)
+
+## Connecting model registry actions to your CICD
+
+As we [noted](/docs/start/model-management/model-registry#GTO-tip) in the model
+registry chapter, all DVC model registry actions are captured in your git
+repository as git tags with a specific format.
+
+This also means that we can create CICD actions in our git repository which will
+be triggered whenever.
+
+In the following, we will have a look at an example CICD workflow on GitHub
+which runs whenever we assign a version of our model to the "test" stage in the
+model registry. The workflow will check whether the model artifact size is less
+than 50 MB and fail if it isn't. To motivate the example, we can imagine that we
+are deploying models to edge devices where storage comes at a premium.
+
+Go to the `.github/workflows/test-model-artifact.yml`. This is the file that
+GitHub uses to run our CICD workflow. You can see
+[runs of this workflow](https://github.com/iterative/example-get-started-experiments/actions/workflows/test-model-artifact.yml)
+in our example repository.
+
+At the beginning of the workflow file you will see this code
+
+```yaml
+on:
+  # the workflow is triggered whenever a tag is pushed to the repository
+  push:
+    tags:
+      - '*'
+```
+
+The code tells GitHub to run the workflow every time a tag is pushed to the
+repository.
+
+This means that the workflow will run whenever we run model registry actions,
+but we also want it to limit to specific ones for our specific workflow. That's
+where our GTO GitHub action comes into play - in the "parse" job of our workflow
+it parses all tags and if they are GTO tags, it gives us the name of the model,
+its version, stage (if any) and the event in the model registry.
+
+This is captured in the "parse" job which you can simply copy and paste into
+most CICD jobs of your own.
+
+```yaml
+# This job parses the git tag with the GTO GitHub Action to identify model registry actions
+parse:
+  runs-on: ubuntu-latest
+  steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: 'Parse GTO tag'
+      id: gto
+      uses: iterative/gto-action@14723404a00bb0c1e759c02ffcd24279df5815c2
```

Comment:
What is this rev?

Reply:
TBH I wonder what it looks like without the `gto-action`. Calling gto directly in the script could arguably be more transparent and definitely would be more helpful for non-github users.

Reply:
I was wondering about that also. It is the one we use in the example repository but it probably should be updated there also.

I think it would make sense to write this also for GitLab and Bitbucket and there you don't have the action. But it makes sense for me to keep it in the GH version. You are right that it is less transparent though and we don't have GL or BB code at hand now, so your suggestion might be the best approach at the moment.

Reply:
For now, I at least replaced the GTO action version with v2

Reply:
I agree that replacing it with calling GTO directly is more transparent, but I want to push an update now.

Reply:
Okay, added https://github.com/iterative/dvc.org/issues/4947